config/config.php

```diff
@@ -308,7 +308,7 @@
         |
         */
 
-      'show_black_list_exception' => env('JWT_SHOW_BLACKLIST_EXCEPTION', true),
+      'show_black_list_exception' => env('JWT_SHOW_BLACKLIST_EXCEPTION', 0),
```

Comment:
in my opinion `'show_black_list_exception' => boolval(env('JWT_SHOW_BLACKLIST_EXCEPTION', false)),` would be better to have a bool value in any case. What's your opionion about that @Messhias ?